sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/SqlStdOperatorMappingTable.java

```diff
@@ -29,127 +29,620 @@
 /** SqlStdOperatorMappingTable. */
 @Internal
 public class SqlStdOperatorMappingTable {
+
+  /**
+   * Keep this list in sync with
+   * https://github.com/google/zetasql/blob/master/zetasql/public/builtin_function.proto. Uncomment
```

Comment:
We have historically avoided the pattern of blocks of commented out constants. I don't have a strong opinion if this is short term, but @kennknowles might.

Reply:
Make sense. I can change it.